configure.ac

```diff
@@ -2946,6 +2946,44 @@ esac
 ],
 [AC_MSG_RESULT(default)])
 
+
+validate_tzpath() {
+    # Check that each element of the tzpath is an absolute path
+    all_paths=$1
+    while test "${all_paths#*:}" != "$1"; do
```

Comment:
The comparison to "$1" confuses me. $all_paths starts out as $1, and you're only removing things, so surely this loop would never end? It also feels like you could do this with grep instead:

  if ( echo $1 | grep -qE '(^|:)([^/]|$)' ); then
      AC_MSG_ERROR(...);
      return 1;
  fi

(There are already other uses of grep -E in configure.ac, so I think it's safe to rely on.)

Reply:
I like it, though it was annoyingly tricky to pull off. It took me *forever* to realize that `autoconf` was stripping out the `[]` characters from the grep string.